tests/test_hello_page.py

```diff
@@ -0,0 +1,32 @@
+"""Tests related to hello page"""
+import logging
+
+from constants import header
+
+
+class TestHelloPage:
+    """Stores tests for create hello page base functionality"""
+
+    log = logging.getLogger("[TestHelloPage]")
+
+    def sign_out_from_hello_page_after_sign_in(self, start_page, random_login):
+        """
+        - Pre-conditions:
+            - Sign In as a user
+        - Steps:
+            - Click on "Sign Out" button
+            - Navigate to start page
+        """
+        # Sign In as a user
+        hello_page = start_page.sign_in(random_login)
+
+        # Verify login success
+        hello_page.verify_sign_up_message(username=random_login.username_value)
+
+        # Click on "Sign Out" button
+        header.sign_out_button()
```

Comment:
What is header ? From where did you get it ?

Reply:
Fixed: reference to hello_page. into the string27:  hello_page.header.my_profile_button() was added